app.config.js

```diff
@@ -13,16 +13,9 @@ configurationFiles.push(
 )
 
 const isDrive = process.env.COZY_APP_SLUG === 'drive'
-const isPhotosServices = process.env.COZY_APP_SLUG === 'photos:services'
 
 if (isDrive) configurationFiles.push(require('./webpack/drive.config.js'))
-else if (isPhotosServices) {
-  configurationFiles.push(
-    require('cozy-scripts/config/webpack.config.services')
```

Comment:
The v1.10.0 of cozy-scripts will add automatically the services config into bundles